src/tag.ts

```diff
@@ -43,6 +45,9 @@ export default class GlideCoreTag extends LitElement {
 
   static override styles = styles;
 
+  @property({ reflect: true })
+  color?: 'green' | 'indigo' | 'red';
```

Comment:
Let me know how you think of `variant`. But I think of it as of naming a change or group of changes for which there's otherwise not a good name. Here, however, we have a name that explicitly describes the state change in question.